tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
```

Comment:
Can we get rid of the comments?

Reply:
It's a JSON5 file, but can't use `.json5` 🤷‍♀️

Reply:
It's an issue with Github. Commented JSON is fine. Even Mr @douglascrockford is cool with it ;)

Reply:
I do not know what JSON5 is, but whatever it is, I am not cool with it.

Reply:
@douglascrockford How about https://hjson.org/? 😄

Reply:
oh my, way to go on using the ad verecundiam in public @mohsen1

Reply:
@douglascrockford Re `I do not know what JSON5 is`:
> The JSON5 Data Interchange Format ([JSON5](https://json5.org/)) is a superset of JSON that aims to alleviate some of the limitations of JSON by expanding its syntax to include some productions from ECMAScript 5.1.